.rubocop_todo.yml

```diff
@@ -680,6 +680,7 @@ Style/FrozenStringLiteralComment:
     - 'lib/datadog/tracing/contrib/sinatra/patcher.rb'
     - 'lib/datadog/tracing/contrib/sinatra/tracer.rb'
     - 'lib/datadog/tracing/contrib/sinatra/tracer_middleware.rb'
+    - 'lib/datadog/tracing/contrib/span_attribute_schema.rb'
```

Comment:
We should address the https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment issue and remove this line from `.rubocop_todo.yml`.

Removing this line and running `bundle exec rubocop -A` will address this specific issue.